x-pack/plugins/fleet/server/services/epm/packages/install.ts

```diff
@@ -97,22 +97,50 @@ export async function ensureInstalledDefaultPackages(
   });
 }
 
+export async function isPackageInstalled(options: {
```

Comment:
can we call this `isPackageVersionInstalled` installed? I was confused as to the difference between this and `getInstallation` before realizing that this one checks if a specific version is installed